index.html

```diff
@@ -5,6 +5,7 @@
         <link rel="stylesheet" href="style.css">
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>img[alt="www.000webhost.com"] {display: none;}</style>
```

Comment:
Why is this here?

Reply:
This is necessary to hide the hosting ads